src/index.ts

```diff
@@ -6,6 +6,8 @@ export * from './Cards/CardManager';
 export * from './Cards/CardVerifier';
 export * from './Cards/RawSignedModel';
 export * from './Cards/ModelSigner';
+export * from './Client/Connection';
+export * from './Client/VirgilAgent';
```

Comment:
I'd really like to avoid exporting these classes. They don't add much value to consumers of SDK but complicate the API and should thus be "internal" in my opinion. There must be other ways to use a different header in e3kit (not sure it's even necessary)

Reply:
I think we don't use one of the main feature of OOP - reusability. All classes can be used independently. And I show the example how I use them in e3kit. `Connection` class could be used in keyknox, but not used because it wasn't exported. And we need to detect keyknox and e3kit also.
I agree that VirgilAgent could not be exported, because it can be encapsulated in `Connection` class.
About consumers, we make products based on Virgil SDK, so why not to export the `classes`which we can reuse. And about complexity of SDK it need to be handled by documentation. We provide high level options for consumers and documented them, so there is official way how to use them. And if class is exported it doesn't mean it need to be used by everyone.